README.md

```diff
@@ -175,10 +175,15 @@ $ chisel server --help
     provide a certificate notification email by setting CHISEL_LE_EMAIL.
 
     --tls-ca, a path to a PEM encoded CA certificate bundle or a directory
-    holding multiple PEM encode CA certificate bundle files, which is used to 
-    validate client connections. The provided CA certificates will be used 
-    instead of the system roots. This is commonly used to implement mutual-TLS. 
-
+    holding multiple PEM encode CA certificate bundle files, which is used to
+    validate client connections. The provided CA certificates will be used
+    instead of the system roots. This is commonly used to implement mutual-TLS.
+
+    --ldap-config, a path to a file containing the ldap authentication
+    configuration.
+    here is an example of an ldap-config file
+    { "BindDN": "CN=ldapUser,OU=Users,OU=example,DC=EXAMPLE,DC=COM", "BindPassword": "ldapUserPassword", "Url": "example.com:636", "BaseDN": "OU=Users,OU=example,DC=EXAMPLE,DC=COM", "Filter": "(&(objectClass=person)(objectClass=user))", "IDMapTo": "sAMAccountName", "CA": "", "Insecure": true }
```

Comment:
this `README.md` is generated using the command in the comment on L108 so this example will get replaced with whats in `main.go`